load-fhir-data.js

```diff
@@ -17,14 +17,39 @@ GC.get_data = function() {
     var hidePatientHeader = (smart.tokenResponse.need_patient_banner === false);
     GC.Preferences.prop("hidePatientHeader", hidePatientHeader);
 
-    var ptFetch = smart.patient.read();
-    var vitalsFetch = smart.patient.api.fetchAll({type: "Observation", query: {code: {$or: ['3141-9', '8302-2', '8287-5', '39156-5', '18185-9', '37362-1', '11884-4']}}});
-    var familyHistoryFetch = smart.patient.api.fetchAll({type: "FamilyMemberHistory"});
+    var patient = smart.context.patient;
+
+    var vitalsFetch = $.Deferred();
+    var familyHistoryFetch = $.Deferred();
+    var ptFetch = patient.read();
+
+    patient.Observation.where.
+    nameIn(['3141-9', '8302-2', '8287-5', '39156-5', '18185-9', '37362-1', '11884-4']).
```

Comment:
unfortunately, we are using the Cerner FHIR service, which is not DSTU2 compliant. Once the service is updated, we plan to upgrade all such calls.